### PR TITLE
Add publish command

### PR DESCRIPTION
**Pull request by RichardLitt**
This adds a publish command, which automatically opens an issue in ipfs/ops-requests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,11 @@ start: build
 dist: build
 	npm run dist
 
+publish: build
+	npm i -g ping-ops-requests
```
Comment (haadcode) [inline Makefile]: Let's install this locally, so without the `-g` flag, in order to not install something unwated globally.

**Comment by haadcode** [comment]
One small comment on the Makefile. Let's add super short section to the readme to describe this process: "run make publish, with a token, it'll open an issue in X and will get deployed by the ops team if all good". Sounds good?

File: Makefile (hunk 1 above)
Comment (RichardLitt) [inline Makefile]: Fixed.

**Comment by RichardLitt** [comment]
Added. Thoughts?

**Comment by haadcode** [comment]
Yeah, looks good! Thank you. 

Before we can merge, I want to be sure how this work, so I'll review it today/tomorrow and merge when good. Looking at it, should be ready to go.

**Comment by haadcode** [comment]
Rebased on master, tweaked the publish command, using the ping-ops-requests fork that allows specifying the issue title.

**Comment by haadcode** [comment]
@RichardLitt I finally got to this. I made some changes (see above) and left my fork of ping-ops-requests in the merged package.json. Once the original module has been updated with the title flag, we can update package.json to use that.

Thank you for making this happen! :) <3

**Comment by RichardLitt** [comment]
@haadcode Updated module. Sweet.

**Comment by haadcode** [comment]
Thanks @RichardLitt, will pull that in.

**Pull request merged**